ddtrace/api.py

```diff
@@ -25,7 +25,12 @@ def __init__(self, hostname, port, headers=None, encoder=None):
 
         # overwrite the Content-type with the one chosen in the Encoder
         self._headers = headers or {}
-        self._headers.update({'Content-Type': self._encoder.content_type})
+        self._headers.update({
+            'Content-Type': self._encoder.content_type,
+            'Meta-Lang': 'python',
+            'Meta-Lang-Version': PYTHON_VERSION,
+            'Meta-Lang-Interpreter': PYTHON_INTERPRETER,
```

Comment:
I think we should add also `Datadog-Meta-Tracer-Version` that points to the current `dd-trace-py` version. I think you can:
```python
from ddtrace import __version__
self._headers.update({
    # ...
    'Datadog-Meta-Tracer-Version': __version__,
})
```